lib/game.rb

```diff
@@ -0,0 +1,50 @@
+require '../lib/interface.rb'
+
+class Game
+  include Interface
+  attr_accessor :symbol
+
+  def initialize(board, player_1, player_2)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+    @turn = 1
+   end
+
+  def win?(item)
+    temp = []
+    @board.board.each_with_index do |value, index|
+      temp << index if value == item
+    end
+    @board.win_patterns.each do |arr|
+      result = arr - temp
+      return true if result.empty?
+    end
+    false
+  end
+
+
+	def gameplay
+    change_turn until win?(@player_1.symbol) || win?(@player_2.symbol) || @board.full?
```

Comment:
We can avoid the repetition in the `change_turn` method (which I already pointed out in my previous review). 

You can simply call the `action` method directly here in `gameplay`:

```
def gameplay
  until is_game_over?
    action(player_1)
    action(player_2)
  end
  winner
end
```
And then have a method that checks if a game is already finished:

```
def is_game_over?
  win?(@player_1.symbol) || win?(@player_2.symbol) || @board.full?
end
```
This makes the code cleaner and you have less methods to test. Plus it makes your methods easier to test later on.